### PR TITLE
Update requirements.rst

### PR DESCRIPTION
**Pull request by TriformDaniel**
- 2 packages, right? or do pum & pirogue come in one?
- no need for pum&pirogue for (initial) installation according to @m-kuhn

### CODE REVIEW

File: en/installation-guide/requirements.rst [hunk 1]
```diff
@@ -3,7 +3,8 @@
 Requirements
 =======================
 
-The following Python package (installed using `pip <https://pypi.org/project/pip/>`_) are required:
+The following Python packages (installed using `pip <https://pypi.org/project/pip/>`_) are required:
```
Comment (m-kuhn) [inline en/installation-guide/requirements.rst]: Looks good

File: en/installation-guide/requirements.rst [hunk 2]
```diff
@@ -3,7 +3,8 @@
...
 
 * `PUM <https://github.com/opengisch/pum>`_ for model upgrades
-* `Pirogue <https://github.com/opengisch/pirogue>`_ for model generation and upgrades
\ No newline at end of file
+* `Pirogue <https://github.com/opengisch/pirogue>`_ for model generation and upgrades
+Note: not needed for the installation, see https://github.com/QGEP/docs/issues/107#issuecomment-572481141
```
Comment (m-kuhn) [inline en/installation-guide/requirements.rst]: I think we want people to install these (for a long term stable running of QGEP). I think that this link will be more confusing than helpful in the future once the installation is clearly documented, no?

Comment (TriformDaniel) [reply en/installation-guide/requirements.rst]: ok for me - so leave it out
Your comment just helped me not to get stuck before even starting the installation.

Comment (m-kuhn) [reply en/installation-guide/requirements.rst]: Maybe let's put it like this?

```
.. note::

  These modules are not required for the initial installation but only for subsequent datamodel upgrades.
```

Comment (m-kuhn) [inline en/installation-guide/requirements.rst]:
```suggestion

.. note::

  These modules are not required for the initial installation but only for subsequent datamodel upgrades.
  
```

Comment (m-kuhn) [reply en/installation-guide/requirements.rst]: @TriformDaniel please click on the Commit suggestion button if you like this

**Comment by m-kuhn** [comment]
thank you

**Pull request merged**